cdap-ui/app/directives/c3/charts.js

```diff
@@ -113,6 +119,11 @@ ngC3.directive('c3Line', function () {
   return angular.extend({
     link: function (scope, elem, attr) {
       scope.initC3(elem, 'line', attr, {xtickcount: 5});
+      scope.clearChart = function() {
```

Comment:
Cool. We don't need this function at all. Have removed this in this PR (https://github.com/caskdata/cdap/pull/2707).